components/copyright.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Copyright = () => {
+  const d: number = new Date().getFullYear();
```

Comment:
Instead of using the variable name this way like `d`, we should give a meaningful name like `year`.